racs.go

```diff
@@ -1317,6 +1326,28 @@ func handleProjectBuild(w http.ResponseWriter, r *http.Request, u *user, params
 	w.Write([]byte("OK"))
 }
 
+func handleTaskStop(w http.ResponseWriter, r *http.Request, u  *user, params map[string]string) {
```

Comment:
I think this function should use checkLogin to require users to log in before stopping a task. Since this function isn't called using a form submit, the redirect after login won't work correctly (we need a different mechanism for this later) but at least we can prevent non-logged in users from stopping tasks.

Reply:
Added checkLogin(), but login page not getting rendered in case user not logged in.